cogs/customreactions.py

```diff
@@ -0,0 +1,1111 @@
+                        else:
+                            title = "Option successfully modified! Returning to list of current reactions..."
+                        embed = discord.Embed(title=title)
+                        embed.set_footer(text="Modified by {}.".format(user),
+                                         icon_url=user.avatar_url)
+                        await message.edit(embed=embed)
+                        conn.close()
+                        time.sleep(5)
+
+                elif reaction.emoji == "1️⃣":
+                    if anywhere == 1:
+                        if proposals:
+                            title = "Successfully kept current option! Returning to list of reaction proposals..."
+                        else:
+                            title = "Successfully kept current option! Returning to list of current reactions..."
+                        embed = discord.Embed(title=title)
+                        embed.set_footer(text="Modified by {}.".format(user),
+                                         icon_url=user.avatar_url)
+                        await message.edit(embed=embed)
+                        conn.close()
+                        time.sleep(5)
+                    else:
+                        t = (1, custom_react_id)
+                        c.execute(
+                            'UPDATE CustomReactions SET Anywhere = ? WHERE CustomReactionID = ?',
+                            t)
+                        conn.commit()
+                        self.rebuild_reaction_list()
+                        self.rebuild_proposal_list()
+                        if proposals:
+                            title = "Option successfully modified! Returning to list of current reaction proposals..."
+                        else:
+                            title = "Option successfully modified! Returning to list of current reactions..."
+                        embed = discord.Embed(title=title)
+                        embed.set_footer(text="Modified by {}.".format(user),
+                                         icon_url=user.avatar_url)
+                        await message.edit(embed=embed)
+                        conn.close()
+                        time.sleep(5)
+                elif reaction.emoji == "⏹":
+                    await leave(message)
+                    return
+
+            if reaction.emoji == "5️⃣":
+                embed = discord.Embed(title="Loading...")
+                await message.edit(embed=embed)
+                if proposals:
+                    title = "Modify a custom reaction proposal. React with the option you want"
+                    footer = "{} is currently modifying a custom reaction proposal. \n".format(
+                        user)
+                else:
+                    title = "Modify a custom reaction. React with the option you want"
+                    footer = "{} is currently modifying a custom reaction. \n".format(
+                        user)
```

Comment:
duplicated string literals – use variables and format as needed

Reply:
See https://github.com/idoneam/Canary/pull/273#issuecomment-656869764